pos_stock_available/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Pos Stock Available',
+    'version': '11.0.0.0.1',
+    'category': 'Point of Sale',
+    'author': 'Serpent Consulting Services Pvt. Ltd.',
+    'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+    'website': 'https://www.serpentcs.com',
+    'summary': 'POS on hand stock visibility on product image',
+    'description': 'POS on hand stock visibility on product image',
+    'depends': ['point_of_sale'],
+    'data': ['views/templates.xml',
+    ],
+    'qweb': ['static/src/xml/pos_stock_available.xml'],
+    'installable': True,
+    'auto_install':False,
+    'application':False,
+    'price': 39,
+    'currency': 'EUR',
```

Comment:
This is a paid module, this is crime to put it online. ACT ASAP to remove the module from PR